corehq/motech/fhir/const.py

```diff
@@ -0,0 +1,4 @@
+FHIR_VERSION_4_0_1 = '4.0.1'
+FHIR_VERSIONS = [
+    (FHIR_VERSION_4_0_1, 'R4'),
```

Comment:
how will we handle differences here, since it is bound to the client library version? or is that upcoming?

Reply:
It seems the models in the client library [are generated](https://github.com/smart-on-fhir/client-py/blob/master/fhirclient/models/patient.py#L4).

The [code to generate them is here](https://github.com/smart-on-fhir/fhir-parser). Maybe we should generate the models we need instead of using the client library, so that we can version them, and drop the tightly-coupled dependency.

Reply:
The new commits "Drop SMART on FHIR Python Client" to "`get_json_schema()` and tests" replace this approach in favor of using the JSON schema files from the published spec for a FHIR release.

Future work will use the JSON schema to validate resource instances. We can use the list of resource types to populate the dropdown box to map **case types** in the Data Dictionary.

We _might_ want to use the JSON schema to populate dropdown boxes for mapping **case properties** in the Data Dictionary, but I think that will be impractical. We could potentially use it for type-ahead suggest. (Seems out of scope for initial deliverables.)

Reply:
This is great!